src/PrestaShopBundle/Twig/Component/Toolbar.php

```diff
@@ -70,8 +70,28 @@ public function mount(): void
                 }
             }
 
-            $this->breadcrumbs = $this->menuBuilder->convertTabsToBreadcrumbLinks($tab, $ancestorsTab);
-            $this->hookDispatcher->dispatchWithParameters('actionAdminBreadcrumbModifier', ['tabs' => $tabs, 'breadcrumb' => &$this->breadcrumbs]);
+            $this->setBreadcrumbs($tab, $ancestorsTab, $tabs);
+            $this->setTitle($layoutTitle);
         }
     }
+
+    private function setTitle(string $layoutTitle): void
+    {
+        if (empty($layoutTitle)) {
+            $this->title = $this->breadcrumbs['tab']->name;
+        } else {
+            $this->title = $layoutTitle;
+        }
+    }
+
+    private function setBreadcrumbs(Tab $tab, array $ancestorsTab, array $tabs): void
+    {
+        $this->breadcrumbs = $this->menuBuilder->convertTabsToBreadcrumbLinks($tab, $ancestorsTab);
+        $this->hookDispatcher->dispatchWithParameters('actionAdminBreadcrumbModifier', ['tabs' => $tabs, 'breadcrumb' => &$this->breadcrumbs]);
+    }
+
+    public function getTable(): string
+    {
+        return $this->context->getContext()->controller->table;
```

Comment:
Seems like this field is only correctly set in legacy controllers that extend `AdminController` so there are chances that it's always wrong in migrated pages as it will always contain the default value `configuration`

If it's always wrongly set chances are it's actually useless for migrated pages so maybe we should remove it instead. It may become useful when we display a legacy page in the Symfony layout but then it should be handled as a specific case in those pages, not as a generic thing for all pages.

Reply:
@jolelievre this was mainly used in the ID fields of HTML tags. Removing it breaks the majority of UI tests, because the selector is no longer correct. Perhaps the selectors need to be adapted?